package.json

```diff
@@ -365,9 +365,10 @@
     ]
   },
   "resolutions": {
-    "@typescript-eslint/eslint-plugin": "6.21.0",
-    "@typescript-eslint/parser": "6.21.0",
-    "@typescript-eslint/utils": "6.21.0",
+    "@storybook/react-docgen-typescript-plugin": "1.0.6--canary.9.0c3f3b7.0",
+    "@typescript-eslint/eslint-plugin": "7.18.0",
+    "@typescript-eslint/parser": "7.18.0",
+    "@typescript-eslint/utils": "7.18.0",
```

Comment:
the time has come !

we don't need resolutions anymore. this will also nicely showcase if someone would introduce incompatible lint plugins to our monorepo in future.

```suggestion
```

you need also bump `"@rnx-kit/eslint-plugin": "^0.7.2",` https://github.com/microsoft/rnx-kit/blob/main/packages/eslint-plugin/CHANGELOG.md#070

Reply:
aight! thanks ❤️